diff --git a/disk-utils/fdisk-list.c b/disk-utils/fdisk-list.c
index d0d752e..538709a 100644
--- a/disk-utils/fdisk-list.c
+++ b/disk-utils/fdisk-list.c
@@ -44,7 +44,7 @@ void list_disk_geometry(struct fdisk_context *cxt)
 					   | SIZE_SUFFIX_3LETTER, bytes);
 
 	color_scheme_enable("header", UL_COLOR_BOLD);
-	fdisk_info(cxt,	_("Disk %s: %s, %ju bytes, %ju sectors"),
+	fdisk_info(cxt,	_(UL_COLOR_BOLD_YELLOW"Disk %s: %s"UL_COLOR_BOLD_RESET", %ju bytes, %ju sectors"),
 			fdisk_get_devname(cxt), strsz,
 			bytes, (uintmax_t) fdisk_get_nsectors(cxt));
 	color_disable();
